Makefile

```diff
@@ -12,13 +20,12 @@ lint : ## Static code analysis with Pylint
 
 .PHONY : unit_test
 unit_test : ## Unit tests execution with coverage and xml reports
-	python -m coverage run tests_runner.py unit
-	python -m coverage xml -o coverage.xml
-	python -m coverage html -d coverage
+	pytest $(PYTEST_ARGS) --ignore=climada/test climada/
 
 .PHONY : install_test
 install_test : ## Test installation was successful
-	python tests_install.py report
+	pytest $(PYTEST_JUNIT_ARGS) climada/engine/test/test_cost_benefit.py \
```

Comment:
Note for the future: we should probably make a better install test. If there is already one around, I would propose to switch now.

Reply:
Do you have a particular integration test in mind that would be suitable for that?